pkg/kubectl/cmd/get/get.go

```diff
@@ -210,6 +210,12 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 	if err != nil {
 		return err
 	}
+	// For kubectl get events Added default sorting by lastTimestamp
+	if sortBy == "" {
+		if len(args) == 1 && args[0] == "events" {
```

Comment:
Will this work for alias' for events? -- e.g. "event".

Reply:
Now it will handle both `events` and `event`